### ISSUE TITLE
Printing-plugin

### ISSUE THREAD

**Issue by habella** [kartoza/docker-geoserver#523]
### What is the bug or the crash?

I am having problems using the print plugin.

### Steps to reproduce the issue

1. Go to https://localhost:8080/geoserver/pdf/info.json

### Versions

kartoza/geoserver:2.23.0

**Comment by NyakudyaA** [comment]
Please elaborate further so that you can get assistance. This is a community projects and questions like these discourage people from answering/attending to issues

**Comment by habella** [comment]
The problem is that when entering the address https://localhost:8080/geoserver/pdf/info.json it shows the following error message.

![image](https://user-images.githubusercontent.com/92644432/233358637-16212f4d-ac61-4c21-b5de-bd1716622660.png)

As far as I have been able to search it is related to this post: [ https://www.mail-archive.com/search?l=geoserver-users@lists.sourceforge.net&q=subject:%22%5C%5BGeoserver%5C-users%5C%5D+geoserver%22&o=newest&f=1]( https://www.mail-archive.com/search?l=geoserver-users@lists.sourceforge.net&q=subject:%22%5C%5BGeoserver%5C-users%5C%5D+geoserver%22&o=newest&f=1) 

Not being a Java developer it is more complicated for me to test possible solutions.

On the other hand, at no time I have tried to belittle the work, it is a very good project and I recognize the work being done by the community to maintain it. Sorry if my previous message was misinterpreted.

**Comment by NyakudyaA** [comment]
This is fixed by #569 
You need to create a bash script i.e. `run.sh`
```
#!/bin/bash

wget https://repo1.maven.org/maven2/commons-httpclient/commons-httpclient/3.1/commons-httpclient-3.1.jar \
-O /usr/local/tomcat/webapps/geoserver/WEB-INF/lib/commons-httpclient-3.1.jar
```
Then mount it as a volume
```
      volumes:
        - geoserver-data:/opt/geoserver/data_dir
        - ./run.sh:/docker-entrypoint-geoserver.d/run.sh
```
